config.drupal-xb-dev.yaml

```diff
@@ -5,7 +5,7 @@ web_environment:
   - DB_URL=sqlite://web/sites/default/files/db.sqlite
   - DISPLAY=host.docker.internal:0
   - DRUPAL_TEST_DB_URL=sqlite://web/sites/default/files/db.sqlite
-  - VITE_SERVER_ORIGIN=http://localhost:5173
+  - VITE_SERVER_ORIGIN=https://${DDEV_HOSTNAME}:5174
```

Comment:
Could this have an side effects we need to test for? Does Cypress testing interact with Vite in any way, for example?

Reply:
This is only used when Vite's dev server is running, and Cypress tests are run against the production build. So we don't need to worry about it. 😊